src/main/java/edu/wpi/grip/ui/preview/TextAreaSocketPreviewView.java

```diff
@@ -0,0 +1,28 @@
+package edu.wpi.grip.ui.preview;
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.Socket;
+import javafx.scene.control.TextArea;
+
+/**
+ * A <code>SocketPreviewView</code> that previews sockets by simply calling their <code>toString()</code> method and
+ * showing the result in a text area.
+ */
+public class TextAreaSocketPreviewView<T> extends SocketPreviewView<T> {
+
+    /**
+     * @param eventBus The EventBus used by the application
+     * @param socket   An output socket to preview
+     */
+    public TextAreaSocketPreviewView(EventBus eventBus, Socket<T> socket) {
+        super(eventBus, socket);
+
+        this.setStyle("-fx-pref-width: 20em;");
```

Comment:
So, the .css file is really for defining the style and layout of the program.  The point of this line is to set the number of characters that the textarea shows, and it uses CSS because it's the only way to specify it in ems instead of pixels.  There's no advantage in clarity or maintainability by putting that information in a .css file along with all of the style information.